env/staging/replica_count.yaml

```diff
@@ -20,6 +20,16 @@ spec:
 ---
 apiVersion: apps/v1
 kind: Deployment
+metadata:
+  labels:
+    app: celery
+  name:  celery
+  namespace: notification-canada-ca
+spec:
+  replicas: 3
```

Comment:
I added a section for `celery` as the production file has one but not staging. I assume this is for the desired count of instances.